src/libterm/terminfo/parm.rs

```diff
@@ -144,122 +138,167 @@ pub fn expand(cap: &[u8], params: &[Param], vars: &mut Variables)
                     'l' => if !stack.is_empty() {
                         match stack.pop().unwrap() {
                             Words(s) => stack.push(Number(s.len() as isize)),
-                            _        => return Err("a non-str was used with %l".to_owned())
+                            _ => return Err("a non-str was used with %l".to_owned()),
                         }
-                    } else { return Err("stack is empty".to_owned()) },
+                    } else {
+                        return Err("stack is empty".to_owned())
+                    },
                     '+' => if stack.len() > 1 {
                         match (stack.pop().unwrap(), stack.pop().unwrap()) {
                             (Number(y), Number(x)) => stack.push(Number(x + y)),
-                            _ => return Err("non-numbers on stack with +".to_owned())
+                            _ => return Err("non-numbers on stack with +".to_owned()),
                         }
-                    } else { return Err("stack is empty".to_owned()) },
+                    } else {
+                        return Err("stack is empty".to_owned())
+                    },
                     '-' => if stack.len() > 1 {
                         match (stack.pop().unwrap(), stack.pop().unwrap()) {
                             (Number(y), Number(x)) => stack.push(Number(x - y)),
-                            _ => return Err("non-numbers on stack with -".to_owned())
+                            _ => return Err("non-numbers on stack with -".to_owned()),
                         }
-                    } else { return Err("stack is empty".to_owned()) },
+                    } else {
+                        return Err("stack is empty".to_owned())
```

Comment:
rustfmt now appends these statements with semicolons; may be worthwhile to rerun this?